superset-frontend/src/features/alerts/components/NotificationMethod.tsx

```diff
@@ -24,34 +24,77 @@ import { NotificationMethodOption, NotificationSetting } from '../types';
 import { StyledInputContainer } from '../AlertReportModal';
 
 const StyledNotificationMethod = styled.div`
-  margin-bottom: 10px;
+  ${({ theme }) => `
+    margin-bottom: ${theme.gridUnit * 2.5}px;
 
-  .input-container {
-    textarea {
-      height: auto;
+    .input-container {
+      textarea {
+        height: auto;
+      }
+
+      &.error {
+        input {
+          border-color: ${theme.colors.error.base};
+        }
+      }
+
+      .helper {
+        margin-top: ${theme.gridUnit * 2}px;
+        font-size: ${theme.typography.sizes.s}px;
+        color: ${theme.colors.grayscale.base};
+      }
     }
 
-    &.error {
-      input {
-        border-color: ${({ theme }) => theme.colors.error.base};
+    .inline-container {
+      margin-bottom: ${theme.gridUnit * 2}px;
+
+      > div {
+        margin: ${theme.gridUnit * 0}px;
```

Comment:
If it's 0 it's fine to just put `0px` since that won't really depend on gridUnit's value 😄

Reply:
Thanks! corrected to ```margin: 0px;```

Reply:
Please could you suggest, can I change `role` with `data-testid` . 

like below


```
<span
    data-testid="delbutton"
    tabIndex={0}
    className="delete-button"
    onClick={() => onRemove(index)}
>
```

```
<span
    data-testid="ccbutton"
    tabIndex={0}
    className="delete-button"
    onClick={() => onRemove(index)}
>
```


              
and test case for specific delete button as 

```
const deleteButton = screen.getByTestId('delbutton');
userEvent.click(deleteButton);

```